server/src/test/java/io/crate/metadata/doc/DocSchemaInfoTest.java

```diff
@@ -107,34 +103,11 @@ public String name() {
         docSchemaInfo = new DocSchemaInfo(
             "doc",
             clusterService,
-            nodeCtx,
-            udfService,
             new ViewInfoFactory(() -> null),
             new DocTableInfoFactory(nodeCtx)
         );
     }
 
-    @Test
-    public void testInvalidFunction() throws Exception {
```

Comment:
Moved this as ` test_cannot_create_invalid_function` to `UserDefinedFunctionsTest`

Reply:
Looks like the `udfService` is then no longer needed in this test suite?